components/camel-gson/src/test/resources/org/apache/camel/component/gson/SpringGsonJsonDataFormatTest.xml

```diff
@@ -56,6 +57,17 @@
             <to uri="mock:reversePojo"/>
         </route>
 
+        <route>
+            <from uri="direct:inFormatDate"/>
+            <marshal><custom ref="format_date"/></marshal>
+        </route>
+		
+		<route>
```

Comment:
Would be nice to format this XML correctly

Reply:
Addressed